Library/Homebrew/os/mac/xcode.rb

```diff
@@ -15,7 +15,7 @@ def latest_version
         when "10.8"  then "5.1.1"
         when "10.9"  then "6.2"
         when "10.10" then "7.2.1"
-        when "10.11" then "7.3.1"
+        when "10.11" then "8.0"
```

Comment:
> Presumably from that they're perfectly happy for people to mix & match Xcode/CLT versions going forwards.

I thought we'd been having a bunch of issues resulting from such mix-n-match on El Cap, and that people were being told "don't use Xcode 8 with the 7.3 CLT on El Cap".  Maybe I'm misremembering?  If I'm not, then we will be telling people "don't use Xcode8 with the 7.3 CLT", and also "don't use Xcode 7.3.1 with the 7.3 CLT", which amounts to "don't use the CLT on El_Capitan unless you want to forego Xcode".